integration/common/setup.cfg

```diff
@@ -23,3 +23,21 @@ addopts = -p no:warnings
 
 [mypy]
 ignore_missing_imports = True
+
+[tox:tox]
+envlist = 
+	py3-common-dbt-{1.0,1.3}-parser-{false,true}
+skipsdist = True
+
+[testenv]
+usedevelop = True
+install_command = python -m pip install {opts} --find-links target/wheels/ --find-links ../sql/target/wheels {packages}
+deps = ../../client/python
+	parser-false: -e .[dev_no_parser]
```

Comment:
I think we can remove no-parser option now. We have parser binary wheels on each supported environment.

Reply:
Changed.